pkg/stages/steps_init.go

```diff
@@ -148,6 +148,7 @@ func GetKairosReleaseStage(sis values.System, log types.KairosLogger) []schema.S
 		"KAIROS_RELEASE":        release,
 		"KAIROS_FIPS":           fmt.Sprintf("%t", config.DefaultConfig.Fips),        // Was the image built with FIPS support?
 		"KAIROS_TRUSTED_BOOT":   fmt.Sprintf("%t", config.DefaultConfig.TrustedBoot), // Was the image built with Trusted Boot support?
+		"KAIROS_INIT_VERSION":   values.GetVersion(),                                 // The version of the kairos-init binary
```

Comment:
I was doing the same thing at the same time  :D : https://github.com/kairos-io/kairos-init/commit/0a0861101c4252cd823ac7599afeaf6c623ed1d6

I guess that means I approve the PR. (I'm testing that it actually works before we merge)